docs/docs/guide/usingFreighter.md

```diff
@@ -135,5 +135,44 @@ const userSignTransaction = async (xdr: String) => {
   return signedTransaction;
 };
 
-const userSignedTransaction = userSignTransaction();
+const xdr = ""; // replace this with an xdr string of the transaction you want to sign
+const userSignedTransaction = userSignTransaction(xdr);
+```
+
+freighter-api will return a signed transaction xdr. Below is an example of how you might submit this signed transaction to Horizon using `stellar-sdk` (https://github.com/stellar/js-stellar-sdk):
+
```

Comment:
a user on keybase was unclear about what to do with a signed transaction XDR so I added this bit to spell out what a flow might look like